api/handlers/search/name.go

```diff
@@ -25,11 +26,17 @@ func (n Name) NameSearchHandler(w http.ResponseWriter, r *http.Request) {
 	firstName := r.URL.Query().Get("first_name")
 	lastName := r.URL.Query().Get("last_name")
 	dob := r.URL.Query().Get("dob")
+	userID := r.URL.Query().Get("user_id")
```

Comment:
question: how do we get the `user_id` in the first place? The UI will only be sending `first_name`, `last_name` and/or `dob` and/or `community_id` right?

I think we can remove this line and change the DB search query to handle the logic for this.

Let me know your thoughts on that